vitrina/orgs/views.py

```diff
@@ -54,3 +65,58 @@ def get_context_data(self, **kwargs):
         organization: Organization = self.object
         context_data['members'] = organization.representative_set.all()
         return context_data
+
+
+class RepresentativeCreateView(LoginRequiredMixin, PermissionRequiredMixin, CreateView):
+    model = Representative
+    form_class = RepresentativeCreateForm
+    template_name = 'base_form.html'
+
+    def has_permission(self):
+        return self.request.user.role == Representative.COORDINATOR
+
+    def form_valid(self, form):
+        self.object: Representative = form.save(commit=False)
+        organization_id = self.kwargs.get('organization_id')
+        organization = get_object_or_404(Organization, pk=organization_id)
+        self.object.organization = organization
+        try:
+            user = User.objects.get(email=self.object.email)
+            user.organization = organization
+            user.role = self.object.role
+            user.save()
+
+            self.object.user = user
+            self.object.first_name = user.first_name
+            self.object.last_name = user.last_name
```

Comment:
First and last name should be take from `User`, assuming, that same user does not go with different names in different organizations. These two lines should be deleted.